examples/ExpoMessaging/Readme.md

```diff
@@ -35,4 +35,15 @@
 
      ```bash
      yarn android
-     ```
\ No newline at end of file
+     ```
+
+**Note:**
+
+If your Android build fails please add the following snippet on the `android/gradle.properties` to make it work.
+
+```
+hermesEnabled=true
```

Comment:
Can we explain why we need to add it here to make it work as we've done down below ?

Reply:
Done @vanGalilea